src/api/python/CMakeLists.txt

```diff
@@ -39,4 +39,39 @@ add_library(pycvc4
 target_link_libraries(pycvc4 cvc4 ${PYTHON_LIBRARIES})
 
 python_extension_module(pycvc4)
-install(TARGETS pycvc4 DESTINATION lib)
+
+# Installation based on https://bloerg.net/2012/11/10/cmake-and-distutils.html
+# Create a wrapper python directory and generate a distutils setup.py file
+set(SETUP_PY_IN "${CMAKE_CURRENT_SOURCE_DIR}/setup.py.in")
+set(SETUP_PY    "${CMAKE_CURRENT_BINARY_DIR}/setup.py")
+configure_file(${SETUP_PY_IN} ${SETUP_PY})
+set(PYCVC4_MODULE "${CMAKE_CURRENT_BINARY_DIR}/pycvc4")
```

Comment:
You shouldn't need `${CMAKE_CURRENT_BINARY_DIR}` here since you are already in the `CMAKE_CURRENT_BINARY_DIR`.

Reply:
For some reason that doesn't work for me locally. It looks like it puts it in the `CMAKE_CURRENT_SOURCE_DIR` instead of the binary directory.